### PR TITLE
Evan - Backup Character

### PR DESCRIPTION
**Pull request by ecowden**
H'okay, I know it's premature an' all, but for some reason I have a bad feeling about being stuck in a flammable structure while an army or killer lizards hunts for us. Maybe it's just my incredibly bad aim last Monday, but I just don't have a very good feeling about poor Spark's future. 

Or maybe I was just a wee bit bored and felt like creating a character. We'll never know.

At any rate, I went ahead and created a backup character just for S's and G's.

### CODE REVIEW

File: characters/backup/del.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Nope!
+
+## Background
+
+Age | Height | Weight   | Eyes   | Skin | Hair
+--: | -----: | ------:  | ----   | ---- | ----
+500 | 5'9"   | 115 lbs  | Gray   | Fair | Long, Straight, Brown
+
+* **Personality Traits**. Full of witty aphorisms and has a proverb for every occassion. Most don't make sense.
+* **Ideals**. Community and civilization (was built by beer).
```
Comment (runicfox) [inline characters/backup/del.md]: s/was/were 

Community AND civilization make it a plural subject. Use 'were', not 'was'.

**Comment by runicfox** [comment]
'Monestary' is spelled 'Monastery'. I kinda really dig the Friar Tuck meets Chuck Norris concept!

**Comment by runicfox** [comment]
Also, the crossover with Sparks' clan in Del's background makes me smile.

**Comment by runicfox** [comment]
My only other comment is that technically if Sparks dies, then your replacement character should be the same level as Sparks when he stops... Sparking. So in effect, Del should come in as a 2nd level monk.

**Comment by ecowden** [comment]
Thanks for the spelling and grammar catches!

> I kinda really dig the Friar Tuck meets Chuck Norris concept!

Thanks! You made my day. :wink: 

> Also, the crossover with Sparks' clan in Del's background makes me smile.

 Good eye! I wasn't sure if anyone would catch it, but yes: Del is Sparks's great-great-great-etc-grandfather. :grin:

**Comment by ecowden** [comment]
> My only other comment is that technically if Sparks dies, then your replacement character should be the same level as Sparks when he stops... Sparking. So in effect, Del should come in as a 2nd level monk.

Ooo, good point. I held off leveling up right off the bat because reasons, but I suppose it would make more sense to level him as we go. :+1:

**Comment by runicfox** [comment]
Did you want to merge and close this PR?  I wasn't sure if there was a reason that you were keeping it open.

**Comment by ecowden** [comment]
Yeah, there was a really good reason: I completely forgot... :grin: 

Merging...

**Comment by runicfox** [comment]
No worries!  I'm still trying to muster the energy to write up the Campaign Notes from last time.

**Comment by ecowden** [comment]
Do it! I could use a laugh.  :wink:

**Pull request merged**